nmtwizard/preprocess/preprocess.py

```diff
@@ -228,9 +228,10 @@ def generate_preprocessed_data(self, result='preprocess', preprocess_exit_step=N
                 raise RuntimeError('%s is not a directory' % result_dir)
 
             # Sample files and write information to a special file structure.
-            all_files, summary = sampler.sample(self._config, data_path)
+            oversample_as_weights = self._config.get('data', {}).get('oversample_as_weights', False)
```

Comment:
Maybe the option name should be a bit more explicit. What do you think of "oversample_with_sentence_weighting"?

Reply:
That's one long option name 😆  but I agree that it is more explicit, so ok.